src/ui/inspector/Properties.vue

```diff
@@ -81,6 +82,9 @@ export default {
                         }, this.item)
                     };
                 });
+        },
+        singleSelectNonObject() {
+            return Object.keys(this.item).length === 0 && !this.multiSelect;
```

Comment:
Let's use !this.item.identifier to determine that the selected thing is not a domainObject.